src/Native/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(CoreFX)
 
+set(CMAKE_MACOSX_RPATH ON)
```

Comment:
Seems like newer cmakes have this on by default?
http://www.cmake.org/cmake/help/v3.0/policy/CMP0042.html